platform/spec/services/temporal/activities/request_first_page_activity_spec.rb

```
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Temporal::Activities::RequestFirstPageActivity do
+  subject { described_class.new(context) }
+
+  let(:context) { instance_double('Temporal::Activity::Context', logger: logger) }
```

rubocop (1.68.0)

[convention] Use a constant class reference for verified doubles.

(RSpec/VerifiedDoubleReference)